tests/test_google_analytics_bookmarks.py

```diff
@@ -195,22 +190,19 @@ def test_run(self):
                     second_replication_values_sorted = sorted(second_replication_values)
                     self.assertEqual(second_replication_values_sorted, second_replication_values)
 
-                    # Verify the second sync records respect the previous (simulated) bookmark value
+                   
                     for record in second_sync_messages:
                         with self.subTest(record=record):
+                            # Verify the second sync records respect the previous (simulated) bookmark value
                             self.assertGreaterEqual(record[replication_key], simulated_bookmark_value)
 
-                    # UNCOMMENT START BUG_TDL-16943
-                    # with self.subTest():
-                    #     # Verify today's date is the max replication-key value for the second sync
-                    #     self.assertEqual(second_sync_messages[-1][replication_key], today,
-                    #                      msg=f"today is present in result: {today in second_replication_values}")
-
-                    # with self.subTest():
-                    #     # Verify today's date is the max replication-key value for the first sync
-                    #     self.assertEqual(first_sync_messages[-1][replication_key], today,
-                    #                      msg=f"today is present in result: {today in first_replication_values}")
-                    # UNCOMMENT END BUG_TDL-16943
+                            # Verify today's date is the max replication-key value for the second sync
```

Comment:
@prijendev Can you please reply to this thread what code change is added here?

Reply:
Added assertions that verify that today's date is the max replication-key value for both the syncs. (Last record's replication key will be bookmarked)